src/services/search/service/search-index.ts

```diff
@@ -36,13 +36,16 @@ export type IndexList = {
 };
 
 /**
+ * `permissions` is included in the response as a convenience to
+ * the client, but it is not a field in GSearchIndex itself.
+ *
  * @see https://docs.globus.org/api/search/reference/index_show/
  */
 export const get = function (
   index_id,
   options?,
   sdkOptions?,
-): Promise<JSONFetchResponse<GSearchIndex>> {
+): Promise<JSONFetchResponse<IndexWithPermissions>> {
```

Comment:
I'm not seeing this reflected in the documentation yet, is it something that is still awaiting release?

Reply:
It's been released, I believe the documentation has lagged behind. I'm totally fine waiting to merge this until the docs have been updated. In fact, I'll go ahead an put in a PR to update them

Reply:
https://github.com/globusonline/docs.globus.org/pull/1759